airflow/example_dags/tutorial_etl_dag.py

```diff
@@ -99,32 +99,32 @@ def load(**kwargs):
         python_callable=extract,
     )
     extract_task.doc_md = """\
```

Comment:
I guess it might require `textwrap.dedent` to be added.

Reply:
Done!